portal-kernel/src/com/liferay/portal/kernel/model/role/RoleConstants.java

```diff
@@ -102,6 +102,10 @@
 
 	public static final String TYPE_SITE_LABEL = "site";
 
+	public static final int[] TYPES_DEPOT_AND_REGULAR = {
```

Comment:
@adolfopa Why is this not a private constant inside PortletConfigurationPermissionsDisplayContext? What's the point of making it an api constant and causing the package version bump?

Reply:
I tried to keep all related constants in the same class, that's the only reason. Feel free to move them to a private class.

Thanks!